xbmc/music/MusicDatabase.cpp

```diff
@@ -314,7 +313,7 @@ void CMusicDatabase::CreateAnalytics()
 
   /* Maintain date new and last modified for songs, albums and artists using triggers
      MySQL triggers cannot modify a table that is already being used by the statement that invoked
-     the trigger (to avoid recursion), but can set NEW column values before insert or update. 
+     the trigger (to avoid recursion), but can set NEW column values before insert or update.
```

Comment:
@lrusak I notice that you are finding and fixing whitespace that running clang format (on Windows 10) doesn't see. That IDE difference is going to be tedious. Can anything be done about it?

Reply:
Visual Studio 2019 can be configured to show white spaces:

![Captura de pantalla 2021-04-25 130004](https://user-images.githubusercontent.com/58434170/115990954-57330080-a5c6-11eb-9d9c-6fa26ceb848a.png)

Reply:
> Although I do not use it and I do not know exactly the details it can automatically remove the white spaces at the end of the lines when saving the files.

That sounds like what I need, so should we not have a Kodi project .editorConfig file?

Reply:
We could look at it... it should be added to .gitignore, etc.

```
[*.cpp]
insert_final_newline = true
trim_trailing_whitespace = true
```

Reply:
Then is need to add .editorconfig to the VS project:

![Captura de pantalla 2021-04-25 133214](https://user-images.githubusercontent.com/58434170/115991877-dde9dc80-a5ca-11eb-8f3b-724bd25d059b.png)